client/my-sites/theme/main.jsx

```diff
@@ -130,12 +133,19 @@ class ThemeSheet extends Component {
 		section: '',
 	};
 
+	state = {};
+
 	scrollToTop = () => {
 		window.scroll( 0, 0 );
 	};
 
 	componentDidMount() {
 		this.scrollToTop();
+
+		const { syncActiveTheme } = this.props;
+		if ( syncActiveTheme ) {
+			this.props.themeStartActivationSync( this.props.siteId, this.props.themeId );
+		}
```

Comment:
It may be worth being a bit defensive against either `siteId` or `themeId` not being defined.
```suggestion
		const { siteId, syncActiveTheme, themeId, themeStartActivationSync } = this.props;
		if ( syncActiveTheme && siteId && themeId ) {
			themeStartActivationSync( siteId, themeId );
		}
```